packages/cli/config/eslint-factory.js

```diff
@@ -131,6 +131,13 @@ function createConfig(dir, extraConfig = {}) {
         2,
         {
           paths: [
+            ...['.', '..', '../..', '../../..', '../../../..', '../../../../..']
```

Comment:
I really want to roll this out broadly, but there will be a lot of occurrences of these for people to fix. We discussed this during the project meeting at KubeCon, and that we overall are having trouble rolling out lint rules right now due to the lack of any kind of deprecation path.

What do you think, should we explore introducing these as warnings instead and making the warnings not fail the linting? Of course people can ignore the warnings, but at least then we give people a proper path to upgrade without immediate breaking changes.

Reply:
Ugh. Having multiple levels for a single rule seems to be poorly supported. There's stuff like this https://tech.devoted.com/emitting-multiple-error-levels-with-the-same-eslint-rule-b5b2f981010d where they make an entire module just to have a clone of the original rule. :/